setup.py

```diff
@@ -11,7 +11,7 @@
         "safetensors",
         "k-diffusion",
         "diffusers==0.15.1",
```

Comment:
Making a note to myself to upgrade diffusers after merging this, and test before pushing to beta